reference/deprecation-notices.md

```diff
@@ -2,16 +2,27 @@
 
 This document contains a list of deprecation notices for Anbox Cloud and its components.
 
+## Node controller
+*Deprecated in 1.23.0* ; *Removal in 1.24.0*
```

Comment:
Is there a reason for this to be `1.24.0` and not `1.24.2` ?

Reply:
@adglkh could you confirm when this will be removed completely?

Reply:
As per SU discussion, this stays as is for node controller. But corrected for appliance v1 implementation as removal in 1.25.0.